scripts/validate_skin.py

```
@@ -0,0 +1,220 @@
+    if isinstance(game.get("start_time_utc"), str):
+        from datetime import datetime
+        game["start_time_utc"] = datetime.fromisoformat(game["start_time_utc"])
+    return game
+
+
+def parse_size(value: str):
+    try:
+        w, h = value.lower().split("x")
+        return int(w), int(h)
+    except ValueError:
+        raise argparse.ArgumentTypeError(f"size must look like 128x32, got {value!r}")
```

Ruff (0.15.21)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)